cfme/control/explorer/alerts.py

```diff
@@ -291,12 +295,28 @@ class AlertCollection(BaseCollection):
 
     def create(self, description, severity=None, active=None, based_on=None, evaluate=None,
             driving_event=None, notification_frequency=None, snmp_trap=None, emails=None,
-            timeline_event=None, mgmt_event=None):
+            timeline_event=True, mgmt_event=None):
+        """ Create a new alert in the UI.
+
+            Note: Since alerts in CFME require a description, driving_event, and one
+            of snmp_trap, emails, timeline_event, or mgmt_event, we set defaults to
+            timeline_event=True, and driving_event=<item_from_dropdown>
+
+            This allows creation of an alert only by a description. e.g.
+            >>> alert = appliance.collections.alerts.create('my_alert_description')
+            >>> alert.delete()
+        """
+        view = navigate_to(self,"Add")
+        if driving_event is None:
+            view.driving_event.select_by_visible_text(partial_match('Hourly'))
```

Comment:
In-line comment as to why hourly is selected here

Reply:
Consider using `all_options` instead of actually selecting and then reading to get the full text.

Reply:
See recent update -- I select the first item from the `all_options` now